sys/include/checksum/crc16_ccitt.h

```diff
@@ -37,8 +34,92 @@
 extern "C" {
 #endif
 
+/**
+ * @brief           Update CRC16-CCITT-KERMIT
+ *
+ * @param[in]  crc  A start value for the CRC calculation, usually the
+ *                  return value of a previous call to
+ *                  crc16_ccitt_kermit_calc(), crc16_ccitt_mcrf4xx_calc() or crc16_ccitt_kermit_update()
+ * @param[in]  buf  Start of the memory area to checksum
+ * @param[in]  len  Number of bytes to checksum
+ *
+ * @return          Checksum of the specified memory area based on the
+ *                  given start value
+ */
+uint16_t crc16_ccitt_kermit_update(uint16_t crc, const unsigned char *buf, size_t len);
+
+/**
+ * @brief           Calculate CRC16-CCITT-KERMIT
+ *                  polynom=0x1021
+ *                  seed=0x0000
+ *                  check=0x2189
+ *
+ * @param[in]  buf  Start of the memory area to checksum
+ * @param[in]  len  Number of bytes to checksum
+ *
+ * @return          Checksum of the specified memory area
+ */
+uint16_t crc16_ccitt_kermit_calc(const unsigned char *buf, size_t len);
+
+/**
+ * @brief           Calculate CRC16-CCITT-MCRF4XX
+ *                  polynom=0x1021
+ *                  seed=0xffff
+ *                  check=0x6f91
+ *
+ * @param[in]  buf  Start of the memory area to checksum
+ * @param[in]  len  Number of bytes to checksum
+ *
+ * @return          Checksum of the specified memory area
+ */
+uint16_t crc16_ccitt_mcrf4xx_calc(const unsigned char *buf, size_t len);
+
+/**
+ * @brief           Update CRC16-CCITT-FALSE
+ *
+ * @param[in]  crc  A start value for the CRC calculation, usually the
+ *                  return value of a previous call to
+ *                  crc16_ccitt_aug_calc(), crc16_ccitt_false_calc() or crc16_ccitt_false_update()
```

Comment:
Of course this needs to be adopted then:
```suggestion
 *                  crc16_ccitt_false_calc() or crc16_ccitt_false_update()
```